src/pages/Search/SearchPage.tsx

```diff
@@ -17,7 +17,9 @@ type SearchPageProps = StackScreenProps<AuthScreensParamList, typeof SCREENS.SEA
 function SearchPage({route}: SearchPageProps) {
     const {shouldUseNarrowLayout} = useResponsiveLayout();
     const styles = useThemeStyles();
+    const {isCustomQuery} = route.params;
```

Comment:
can you please destructure all the params: `isCustomQuery`, `q` and `policyIDs` and use them in line 25?
that way all the params are in one place used the same way